src/client/index.ts

```diff
@@ -1,78 +1 @@
-                const spines: SpineTree[] = [];
-                for (const item of items) {
-                    spines.push(Object.assign(new SpineTree(), item));
-                }
-                return spines;
-                //                    return Object.assign(new SpineTree(), item);
-            }
-        );
-}
-
-export async function loadSpineNode(spineTreeId: string, id: string = 'root'): Promise<SpineNode> {
-    return fetch('/api/v1/cc-proxy/SpineNode/' + id + '?spineTreeId=' + spineTreeId, { headers: { accept: 'application/json' }, method: 'GET' })
-        .then((response: Response): any => response.json())
-        .then(
-            (item: any): SpineNode => {
-                return Object.assign(new SpineNode(), item);
-            }
-        );
-}
-
-export async function loadSkill(id: string = 'root'): Promise<Skill> {
-    return fetch('/api/v1/cc-proxy/Skill/' + id, { headers: { accept: 'application/json' }, method: 'GET' })
-        .then((response: Response): any => response.json())
-        .then(
-            (item: any): Skill => {
-                return Object.assign(new Skill(), item);
-            }
-        );
-}
-
-export async function displaySpineTrees(treeListId: string): Promise<void> {
-    const spineTrees: SpineTree[] = await loadSpineTrees();
-
-    const ul: HTMLElement = document.getElementById(treeListId);
-    ul.className = 'spineTreeList';
-    for (const tree of spineTrees) {
-        const li: HTMLElement = document.createElement('li');
-        li.appendChild(document.createTextNode(`${tree.name} (${tree.id})`));
-        ul.appendChild(li);
-    }
-}
-
-export async function displaySpineTreeElements(elementListId: string, spineTreeId: string, nodeId = 'root') {
-    const spineNode: SpineNode = await loadSpineNode(spineTreeId, nodeId);
-    const ul: HTMLElement = document.getElementById(elementListId);
-    const li: HTMLElement = document.createElement('li');
-    li.appendChild(document.createTextNode(spineNode.name));
-    ul.appendChild(li);
-
-    for (const childId of spineNode.childrenIds) {
-        const innerUl = document.createElement('ul');
-        innerUl.id = 'node-' + childId;
-        innerUl.className = 'spineNodeList';
-        ul.appendChild(innerUl);
-        await displaySpineTreeElements('node-' + childId, '', childId); // No need to convey the snapshotId because it is contained into the child id
-    }
-
-    if (0 < spineNode.skillIds.length) {
-        const innerUl = document.createElement('ul');
-        innerUl.className = 'spineSkillList';
-        ul.appendChild(innerUl);
-        for (const skillId of spineNode.skillIds) {
-            const skill = await loadSkill(skillId);
-            const innerLi = document.createElement('li');
-            innerLi.appendChild(document.createTextNode(skill.name));
-            innerUl.appendChild(innerLi);
-        }
-    }
-}
+export { LearningSpineBrowser, loadSkill } from './components/learning-spine-browser';
```

Comment:
BTW, because you remove the code which is now available from the generated documentation, can you remove the code that was using it? See the commented block into the `index.html` file.